x/cdp/keeper/auctions_test.go

```diff
@@ -92,6 +92,66 @@ func (suite *AuctionTestSuite) TestDebtAuction() {
 	suite.Equal(cs(c("debt", 90000000000)), acc.GetCoins())
 }
 
+func (suite *AuctionTestSuite) TestGetTotalSurplus() {
+	sk := suite.app.GetSupplyKeeper()
+
+	// liquidator account has zero coins
+	suite.Equal(sdk.NewInt(0), suite.keeper.GetTotalSurplus(suite.ctx, types.LiquidatorMacc))
```

Comment:
we should use a require context for the testing suite since the default is an assertion. Applies to all the new tests 👍 

```suggestion
	suite.Require().Equal(sdk.NewInt(0), suite.keeper.GetTotalSurplus(suite.ctx, types.LiquidatorMacc))
```

Reply:
awesome, great tip 💯.  Added suite.Require() to rest of new tests